File: ClientApp/src/app/models/campaign.ts
import { CampaignDetail } from './campaignDetail';
import { Employee } from 'src/app/models/employee';
export interface Campaign {
    id?: number,
    user?: Employee,
    startDate?: number,
    ifinishDated?: number,
    isActive?: boolean,
    campaignDetails?: CampaignDetail[]
}

File: Core/Models/Campaign.cs
using System;
using System.Collections.Generic;
using System.Collections.ObjectModel;

namespace Kaizen.Core.Models
{
    public class Campaign : BaseEntity
    {
        public string UserId { get; set; }
        public ApplicationUser User { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime FinishDate { get; set; }
        public bool IsActive { get; set; }
        public ICollection<CampaignDetail> CampaignDetails { get; set; }

        public Campaign()
        {
            this.CampaignDetails = new Collection<CampaignDetail>();
        }
    }
}